tests/integration/steps/resource.go

```diff
@@ -391,7 +389,13 @@ func ResourceNotPresent(ctx context.Context, kind string) error {
 }
 
 func IstioResourceContainerHasRequiredVersion(ctx context.Context, containerName, kind, resourceName, namespace string) error {
-	requiredVersion := strings.Join([]string{controllers.IstioVersion, controllers.IstioImageBase}, "-")
+	merger := manifest.NewDefaultIstioMerger()
+	version, prerelease, err := manifest.GetIstioVersion(&merger)
+	if err != nil {
+		return err
+	}
+
+	requiredVersion := fmt.Sprintf("%s-%s", version, prerelease)
```

Comment:
We have exactly this formatting in multiple places. I'd say we should avoid this and have it rather returned in that format by a function.